l3/1_notebook.ipynb

```diff
@@ -142,6 +142,7 @@
     "### Command mode <a class=\"anchor\" id=\"command-mode\"></a>\n",
     "\n",
     "You can start any code cell with `!` to run it as a shell command. For example, you can use `!ls` to list the files in the current directory.\n",
+    "If `!` did not work, try to use `%` magic command.\n",
```

Comment:
`!` would work. You can start by `Alternatively,`. And what should follow the % so that magic command runs the cell as bash script?